server/conn.go

```diff
@@ -1128,6 +1128,18 @@ func (cc *clientConn) handleLoadData(ctx context.Context, loadDataInfo *executor
 	// processStream process input data, enqueue commit task
 	go processStream(ctx, cc, loadDataInfo)
 	err = loadDataInfo.CommitWork(ctx)
+	if terror.ErrorEqual(err, executor.ErrQueryInterrupted) {
+		// drain the data from client conn util empty packet received, otherwise the connection will be reset
```

Comment:
Or should we just let the `kill` finish and connection be reset, it may take lots of time reading a large file
@jackysp @imtbkcat

Reply:
What does MySQL do?

Reply:
For MySQL, when killed is received, the processing thread will read all the reamin data, and then report error
```
    if (read_file_from_client)
      read_info.skip_data_till_eof();
```
@jackysp @imtbkcat

Reply:
Got.